### PR TITLE
Document contribution workflow

### LINKED ISSUE #261: Document contribution workflow for project
We currently don't have a defined workflow for someone wanting to contribute to the PathforaJS project. The goal of this card is to document that flow. 

At minimum, should include: 
- Create issues (apply labels, etc)
- PR process (cover rebase, peer reviewing, adding tests, deleting tests)

### PR DESCRIPTION
**Pull request by prosdev**
This PR closes #261. 

An update to the README to document some contribution workflow. This is definitely not intended to be a hard guideline, but to encourage consistency in process. Please feel free to make suggestions/changes, @sarahyablok @markhayden .

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -52,6 +52,45 @@ There are 4 types of modules and 5 layouts currently supported.
...
+
+!["Pull Requests"](https://s3-us-west-2.amazonaws.com/pros-lytics/screenshots/issue261_pr_markdown.png)
+
+PathforaJS leverages Travis CI for continuous integration. As a result, existing tests will run against all the PRs and all branches pushed up to the repo. PRs will not get reviewed until CI is green. Still, it is advisable to run `npm test` on your local branch before creating a PR.
+
+Below is an example of a stale PR, with failing Travis CI. Click the `Details` link to see what's going on with Travis CI.
+
+!["Blocked PR"](https://s3-us-west-2.amazonaws.com/pros-lytics/screenshots/issue261_PRs.png)
+
+### Git notes
```
Comment (sarahyablok) [inline README.md]: We currently done do rebases on pathfora and not sure if we want to. Might be worth having a larger discussion on this.

File: README.md [hunk 2]
```diff
@@ -52,6 +52,45 @@ There are 4 types of modules and 5 layouts currently supported.
   pathfora.initializeWidgets([module]);
   ```
 
+## Communication
+**`slack`** - There’s a [slack channel](https://join.slack.com/t/pathforajs/shared_invite/enQtMjcyNDEzMDY4NzIxLTI5ZDIyMDI2NGEzNjU4NDE3MTgyOWQ2YzM5MzhjZjVmZDljMDE3NmU5MDFmYmExNTA5ODlhZmE4NmM2ZmE3MTI). Feel free to join and collaborate!
+
+## Contributing to Pathfora
```
Comment (sarahyablok) [inline README.md]: I think this is a nicely written doc on contribution, but perhaps deserving of its own markdown file. We could make a `contributing.md` file in the repo and then link it from the README. I feel like the readme's getting a bit long.

Comment (prosdev) [reply README.md]: Ahhh yeah, kind of like what we do with the License section for the other PR. I wholeheartedly agree with this! Will make the change and move it on over.

File: README.md (hunk 1 above)
Comment (prosdev) [reply README.md]: 👍

File: README.md (hunk 2 above)
Comment (prosdev) [reply README.md]: @sarahyablok Updated based on your suggestion.

**Pull request merged**